src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -170,7 +171,8 @@ export default class DefaultDeviceController
   constructor(
     private logger: Logger,
     options?: { enableWebAudio?: boolean },
-    private browserBehavior: ExtendedBrowserBehavior = new DefaultBrowserBehavior()
+    private browserBehavior: ExtendedBrowserBehavior = new DefaultBrowserBehavior(),
+    private eventController?: EventController
```

Comment:
is this it weird to have a private property with get & set? Can this just be public? You can define this property in the interface too so no need for get and setter.

Reply:
Yeah that makes it a lot cleaner.